pytorch_lightning/strategies/ddp.py

```diff
@@ -63,7 +63,10 @@
 if _FAIRSCALE_AVAILABLE:
     from fairscale.optim import OSS
 if _TORCH_GREATER_EQUAL_1_10:
-    from torch.distributed.algorithms.model_averaging.averagers import ModelAverager
+    if torch.distributed.is_available():
```

Comment:
Could be one conditional statement

Reply:
Yep, thanks Kaushik. This is a draft PR, and I've to build PyTorch from source to turn off distributed mode to check if it works. I'll get back to these fixes once done, and mark ready for review once it is.